packages/model/src/event.ts

```diff
@@ -67,12 +67,13 @@ export type EventCreateDataObject = Pick<
   | 'endDateTimeZone'
   | 'status'
   | 'tags'
-  | 'meetingLink'
```

Comment:
The `create` method is being used only here https://github.com/yldio/asap-hub/blob/master/packages/server-common/src/utils/sync-google-event.ts#L96 and it does not send `meetingLink`.